apps/mull-ui/src/app/components/chat-bubble-list/chat-bubble-list.tsx

```diff
@@ -0,0 +1,30 @@
+import { Post } from 'apps/mull-ui/src/generated/graphql';
+import { avatarUrl } from 'apps/mull-ui/src/utilities';
+import React from 'react';
+import ChatBubble from '../../components/chat-bubble/chat-bubble';
+
+const formatDate = (date: Date) => {
+  return `${date.getHours()}:${String(date.getMinutes()).padStart(
+    2,
+    '0'
+  )} ${date.toLocaleDateString()}`;
+};
+
+export const chatBubbleList = ({ data }) => {
+  if (data) {
+    const posts = data.posts as Post[];
+    var chatBubbleList = posts.map((post) => (
+      <ChatBubble
```

Comment:
Since you're rendering a list of the same component, React needs a key to distinguish each element individually. So, you need a key prop for it. The key can be any string that can identify a single element in the list.

Reply:
something like this
```suggestion
      <ChatBubble key={post.id}
```